test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -649,62 +651,288 @@ private void CombineAndTestTreeEnsembles(IDataView idv, IPredictorModel[] fastTr
+                        }
+                        Assert.Equal(score, 0.4 * scores.Sum() / predCount, 5);
+                        Assert.Equal(prob, 1 / (1 + Math.Exp(-score)), 6);
+                        Assert.True(pred == score > 0);
+                    }
+                }
+            }
+            finally
+            {
+                for (int i = 0; i < predCount; i++)
+                    cursors[i].Dispose();
+            }
+        }
+
+        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))] // x86 fails with "Unknown command: 'train'; Format error at (83,3)-(83,4011): Illegal quoting"
+        public void TestEnsembleCombiner()
+        {
+            var dataPath = GetDataPath("breast-cancer.txt");
+            var inputFile = new SimpleFileHandle(Env, dataPath, false, false);
+#pragma warning disable 0618
+            var dataView = ImportTextData.ImportText(Env, new ImportTextData.Input { InputFile = inputFile }).Data;
+#pragma warning restore 0618
+
+            var predictors = new IPredictorModel[]
+            {
+                FastTree.TrainBinary(Env, new FastTreeBinaryClassificationTrainer.Arguments
+                {
+                    FeatureColumn = "Features",
+                    NumTrees = 5,
+                    NumLeaves = 4,
+                    LabelColumn = DefaultColumnNames.Label,
+                    TrainingData = dataView
+                }).PredictorModel,
+                AveragedPerceptronTrainer.TrainBinary(Env, new AveragedPerceptronTrainer.Arguments()
+                {
+                    FeatureColumn = "Features",
+                    LabelColumn = DefaultColumnNames.Label,
+                    NumIterations = 2,
+                    TrainingData = dataView,
+                    NormalizeFeatures = NormalizeOption.No
+                }).PredictorModel,
+                LogisticRegression.TrainBinary(Env, new LogisticRegression.Arguments()
+                {
+                    FeatureColumn = "Features",
+                    LabelColumn = DefaultColumnNames.Label,
+                    OptTol = 10e-4F,
+                    TrainingData = dataView,
+                    NormalizeFeatures = NormalizeOption.No
+                }).PredictorModel,
+                LogisticRegression.TrainBinary(Env, new LogisticRegression.Arguments()
+                {
+                    FeatureColumn = "Features",
+                    LabelColumn = DefaultColumnNames.Label,
+                    OptTol = 10e-3F,
+                    TrainingData = dataView,
+                    NormalizeFeatures = NormalizeOption.No
+                }).PredictorModel
+            };
+            CombineAndTestEnsembles(dataView, "pe", "oc=average", PredictionKind.BinaryClassification, predictors);
+        }
+
+        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))] // x86 fails with "Unknown command: 'train'; Format error at (83,3)-(83,4011): Illegal quoting"
```

Comment:
>// x86 fails with "Unknown command: 'train'; Format error at (83,3)-(83,4011): Illegal quoting" [](start = 83, length = 95)

Can we have issue about this? it's weird exception, I don't get how change of platform can lead to change of reading data.

Reply:
I copied this comment from the other tests in this file, I'm not sure it actually fails on x86.
@tannergooding , is there an issue for this?

Reply:
https://github.com/dotnet/machinelearning/issues/1216